cypress/automated-tests/youth-pass/address-section-required-fields.cy.js

```diff
@@ -28,6 +28,8 @@ describe('Youth Pass Address Section Required Fields', () => {
         cy.get('#form-section-3 > .form-section-buttons > .form-section-next').click();
         
         cy.get('#element116_Apply').click().blur();
+        // TODO: Remove line 32 when program year choice field is removed from SimpliGov
+        cy.get('#element260_Current').click().blur();
```

Comment:
question: Would it be possible to use conditional logic to say, if this element is present, click the current option, otherwise move on? I think that might allow the same test to be valid when run at any time. I'm reluctant to add new tests that we know are going to break in the future…